keyform/management/commands/migrate_old_data.py

```diff
@@ -0,0 +1,107 @@
+import sqlite3
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.utils.timezone import make_aware, utc
+from keyform.models import *
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        with transaction.atomic():
+            c = sqlite3.connect('/tmp/keyform.db').cursor()
```

Comment:
Just a thought, pass in the database path as an argument. Although it `should` only be run once in production.